<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>메모장</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>편집</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionAllSelect"/>
   </widget>
   <widget class="QMenu" name="menuSee">
    <property name="title">
     <string>보기</string>
    </property>
    <addaction name="actionTools"/>
    <addaction name="actionStatusbar"/>
    <addaction name="actionSideScreen"/>
    <addaction name="actionDownScreen"/>
    <addaction name="separator"/>
    <addaction name="actionFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>검색</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionNextSearch"/>
    <addaction name="actionPrevSearch"/>
    <addaction name="separator"/>
    <addaction name="actionChange"/>
    <addaction name="separator"/>
    <addaction name="actionMove"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>도구</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDocu">
    <property name="title">
     <string>문서</string>
    </property>
    <addaction name="actionAllSave"/>
    <addaction name="actionAllClose"/>
    <addaction name="separator"/>
    <addaction name="actionPrevMemo"/>
    <addaction name="actionNextMemo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>도움말</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSee"/>
   <addaction name="menuSearch"/>
   <addaction name="menuTools"/>
   <addaction name="menuDocu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionTestbutton"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/new.png</normaloff>
     <disabledoff>:/images/images/new_dis.png</disabledoff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>새로 만들기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/open.png</normaloff>
     <disabledoff>:/images/images/open_dis.png</disabledoff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/save.png</normaloff>
     <disabledoff>:/images/images/save_dis.png</disabledoff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>종료</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/saveas.png</normaloff>
     <disabledoff>:/images/images/saveas_dis.png</disabledoff>:/images/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>다른 이름으로 저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/close.png</normaloff>
     <disabledoff>:/images/images/close_dis.png</disabledoff>:/images/images/close.png</iconset>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/undo.png</normaloff>
     <disabledoff>:/images/images/undo_dis.png</disabledoff>:/images/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>실행 취소</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/redo.png</normaloff>
     <disabledoff>:/images/images/redo_dis.png</disabledoff>:/images/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>다시 실행</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/cut.png</normaloff>
     <disabledoff>:/images/images/cut_dis.png</disabledoff>:/images/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>잘라내기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/copy.png</normaloff>
     <disabledoff>:/images/images/copy_dis.png</disabledoff>:/images/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>복사</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/paste.png</normaloff>
     <disabledoff>:/images/images/paste_dis.png</disabledoff>:/images/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>붙여넣기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>삭제</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionAllSelect">
   <property name="text">
    <string>모두 선택</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>찾기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNextSearch">
   <property name="text">
    <string>다음 찾기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionPrevSearch">
   <property name="text">
    <string>이전 찾기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>바꾸기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>줄로 이동</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>도구 모음</string>
   </property>
  </action>
  <action name="actionStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>상태 표시줄</string>
   </property>
  </action>
  <action name="actionSideScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>가장자리 창</string>
   </property>
  </action>
  <action name="actionDownScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>아래 창</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>전체 화면</string>
   </property>
  </action>
  <action name="actionAllSave">
   <property name="text">
    <string>모두 저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionAllClose">
   <property name="text">
    <string>모두 닫기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionPrevMemo">
   <property name="text">
    <string>이전 문서</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+PgDown</string>
   </property>
  </action>
  <action name="actionNextMemo">
   <property name="text">
    <string>다음 문서</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+PgUp</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>차례</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>정보</string>
   </property>
  </action>
  <action name="actionTestbutton">
   <property name="text">
    <string>Testbutton</string>
   </property>
  </action>
  <action name="actionTestbutton2">
   <property name="text">
    <string>Testbutton2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
